src/taxonomy/TaxonomyLayout.jsx

```diff
@@ -1,32 +1,52 @@
+// @ts-check
 import React, { useMemo, useState } from 'react';
 import { StudioFooter } from '@edx/frontend-component-footer';
+import { useIntl } from '@edx/frontend-platform/i18n';
 import { Outlet, ScrollRestoration } from 'react-router-dom';
 import { Toast } from '@edx/paragon';
 
+import AlertMessage from '../generic/alert-message';
 import Header from '../header';
 import { TaxonomyContext } from './common/context';
+import messages from './messages';
 
 const TaxonomyLayout = () => {
+  const intl = useIntl();
   // Use `setToastMessage` to show the toast.
   const [toastMessage, setToastMessage] = useState(null);
+  // Use `setToastMessage` to show the alert.
+  const [alertProps, setAlertProps] = useState(null);
 
   const context = useMemo(() => ({
-    toastMessage, setToastMessage,
+    toastMessage, setToastMessage, alertProps, setAlertProps,
   }), []);
 
   return (
     <TaxonomyContext.Provider value={context}>
       <div className="bg-light-400">
         <Header isHiddenMainMenu />
+        { alertProps && (
+          <AlertMessage
+            data-testid="taxonomy-alert"
+            className="mb-0"
+            dismissible
+            closeLabel={intl.formatMessage(messages.taxonomyDismissLabel)}
+            onClose={() => setAlertProps(null)}
+            // @ts-ignore ToDo: fix object spread type error
+            {...alertProps}
```

Comment:
Is this todo intentional? Is it intended to be fixed in a future PR?

Reply:
No. I was having trouble fixing this type of check before. It was happening because I was missing the state type.
Fixed bcbd4909877f9dd2f62d08e664e7cb4427545b15